packages/containers/psammead-timestamp-container/src/__snapshots__/index.test.jsx.snap

```diff
@@ -30,10 +30,10 @@ exports[`Timestamp should add prefix and suffix 1`] = `
 
 <time
   className="c0"
-  dateTime="2018-10-19"
+  dateTime="۲۰۱۸-۱۰-۱۹"
```

Comment:
`dateTime` should always be using these numerals `"2018-10-19"`. It's because this field is used by search engines. We should not change this field for Persian and other services.